assets/src/edit-story/elements/background/layer.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const LayerBackground = styled.span`
+	opacity: .5;
+`;
+
+function BackgroundLayerContent() {
+	return (
+		<LayerBackground>
+			{ __( 'Background (locked)', 'amp' ) }
```

Comment:
Should be `web-stories` instead of `amp` (there are a few more of these in the PR).

Reply:
Fixed in 0d0b920.